omod/src/main/java/org/openmrs/module/htmlformentryui/page/controller/htmlform/ViewEncounterWithHtmlFormPageController.java

```diff
@@ -26,6 +28,19 @@ public void get(@RequestParam("encounter") Encounter encounter,
 
         patient.setPatient(encounter.getPatient());
 
+        AdministrationService administrationService = Context.getAdministrationService();
```

Comment:
Can this be injected with `@SpringBean` too?

Reply:
No, I tried, and it was an "empty" aop proxy (e.g. all internal references, like the DAO ref were null). @Autowired also resulted in an immediate null reference. I think this is due to the way the application context and HFE service are used/setup, but I'd rather not sink time into investigating or changing any of that in this PR. If it needs to be fixed, it should be it's own ticket (and can't just be a unit test since the application context for testing may not match the runtime application context).

Reply:
change made in latest commit